web-client/src/presenter/actions/takePathForDocumentPreviewAction.test.js

```diff
@@ -0,0 +1,50 @@
+import { applicationContextForClient as applicationContext } from '../../../../shared/src/business/test/createTestApplicationContext';
+import { presenter } from '../presenter-mock';
+import { runAction } from 'cerebral/test';
+import { takePathForDocumentPreviewAction } from './takePathForDocumentPreviewAction';
+
+describe('takePathForDocumentPreviewAction', () => {
+  presenter.providers.applicationContext = applicationContext;
+  presenter.providers.path = {
+    documentInS3: jest.fn(),
+    no: jest.fn(),
+    pdfInMemory: jest.fn(),
+  };
+
+  it('should return path.pdfInMemory when props.fileFromBrowserMemory is defined', async () => {
+    await runAction(takePathForDocumentPreviewAction, {
+      modules: {
+        presenter,
+      },
+      props: {
+        fileFromBrowserMemory: {},
+      },
+    });
+
+    expect(presenter.providers.path.pdfInMemory).toHaveBeenCalled();
+  });
+
+  it('should return path.pdfInS3 when props.documentInS3 is defined', async () => {
```

Comment:
Should `path.pdfInS3` be `path.documentsInS3`?

Reply:
Yes!

Reply:
Updated!